lib/api/controllers/adminController.js

```diff
@@ -84,15 +80,20 @@ class AdminController extends BaseController {
   resetCache (request) {
     assertArgsHasAttribute(request, 'database');
 
-    const
-      database = request.input.args.database,
-      cacheEngine = this.kuzzle.services.list[database];
+    const database = request.input.args.database;
+
+    let cacheEngine;
 
-    if (!cacheEngine || typeof cacheEngine.flushdb !== 'function') {
+    // @todo allow only publicCache
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)